src/styles/dayPicker.scss

```diff
@@ -0,0 +1,309 @@
+:root {
```

Comment:
This looks like its brought in from their code. Is there a way we can import it directly? It's quite bit for us to maintain.